src/internal/module/utils.go

```diff
@@ -0,0 +1,8 @@
+package module
+
+import "fmt"
+
+func GetRepositoryUrl(module Module) string {
+	repoName := fmt.Sprintf("terraform-%s-%s", module.System, module.Name) // TODO hashi?
+	return fmt.Sprintf("https://github.com/%s/%s", module.Namespace, repoName)
+}
```

Comment:
**(nitpick)** It may be a matter of taste, but I'd rather attach the function as a method of `Module`.

```suggestion
type Module struct {
	Namespace string // The module namespace
	Name      string // The module name
	System    string // The module system
}

func (m Module) RepositoryUrl() string {
	repoName := fmt.Sprintf("terraform-%s-%s", module.System, module.Name) // TODO hashi?
	return fmt.Sprintf("https://github.com/%s/%s", module.Namespace, repoName)
}
```

It's also enable for polymorphism as both `Module` and `Provider` will implement this method.

**(nitpick)** Another small nit, I believe that Go-like standard would be 🙂 

```suggestion
func GetRepositoryURL(module Module) string {
	repoName := fmt.Sprintf("terraform-%s-%s", module.System, module.Name) // TODO hashi?
	return fmt.Sprintf("https://github.com/%s/%s", module.Namespace, repoName)
}
```

Reply:
Changed in 020a13aba04c1647c8e0b38c7784f170e10af409